### DISCUSSION TITLE
压测问题

### DISCUSSION THREAD

**Discussion by ghost** [emqx/emqx#6266] [Q&A] [answered]
我在自己的mac上压测8c32g服务器，最多只能到2000多的connections，后续直接2000多都没办法压到，已经按照官网上的系统优化对服务器进行了一遍优化还是不行，实在不清楚什么原因了，麻烦各位大佬帮忙看看什么问题～～感谢🙏
![image](https://user-images.githubusercontent.com/67812498/142966819-a82ed3c3-2dc2-4d1c-9f60-9f82a053d42d.png)

**Comment by qzhuyan** [comment]
Could you describe your setup? vm size, emqx versions and which loadgen do you use? 

2000 sounds like a system limit on max open files.

**Reply by ghost** [reply]
Sorry, I had resolved this problem.

**Comment by ghost** [comment, marked as answer]
伙计们
1. emqx一定要在内网环境下压测，否则你就是在压测WIFI路由器的性能；
2. 还有熟读emqx的官方文档，这样愚蠢的错误就不会再犯了😄
